### PR TITLE
Fix ipv6 netmask

### LINKED ISSUE #825: Incorrect netmask for ipv6
File: [engine/userguide/networking/default_network/ipv6.md](https://docs.docker.com/engine/userguide/networking/default_network/ipv6/)
This line is below one of the earlier examples:
"In this setup the subnet 2001:db8:23:42::/80 with a range from 2001:db8:23:42:0:0:0:0 to 2001:db8:23:42:0:ffff:ffff:ffff is attached to eth0, with the host listening at ..."

Shouldn't it be a ::/64 network that was attached to the eth0? As per the Figure?

### PR DESCRIPTION
**Pull request by shubheksha**
### Related issues (optional)
Fixes #825

### CODE REVIEW

**Comment by JimGalasyn** [comment]
@shubheksha , can you please also fix the image, as  Alessandro suggests? Thanks!

File: engine/userguide/networking/default_network/ipv6.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ on the host:
 
 ![](images/ipv6_slash64_subnet_config.svg)
 
-In this setup the subnet `2001:db8:23:42::/80` with a range from
+In this setup the subnet `2001:db8:23:42::/64` with a range from
 `2001:db8:23:42:0:0:0:0` to `2001:db8:23:42:0:ffff:ffff:ffff` is attached to
```
Comment (aboch) [inline engine/userguide/networking/default_network/ipv6.md]: The range also has to change to
`[...] to 2001:db8:23:42:ffff:ffff:ffff:ffff`

**Comment by johndmulhausen** [comment]
@shubheksha - You're almost there! See @aboch's comment

**Comment by shubheksha** [comment]
Fixed it :D

File: engine/userguide/networking/default_network/ipv6.md [hunk 2]
```diff
@@ -92,11 +92,11 @@ on the host:
...
 ![](images/ipv6_slash64_subnet_config.svg)
 
-In this setup the subnet `2001:db8:23:42::/80` with a range from
-`2001:db8:23:42:0:0:0:0` to `2001:db8:23:42:0:ffff:ffff:ffff` is attached to
+In this setup the subnet `2001:db8:23:42::/64` with a range from
+`2001:db8:23:42:0:0:0:0` to `2001:db8:23:42:ffff:ffff:ffff:ffff` is attached to
 `eth0`, with the host listening at `2001:db8:23:42::1`. The subnet
 `2001:db8:23:42:1::/80` with an address range from `2001:db8:23:42:1:0:0:0` to
-`2001:db8:23:42:1:ffff:ffff:ffff` is attached to `docker0` and will be used by
+`2001:db8:23:42:ffff:ffff:ffff:ffff` is attached to `docker0` and will be used by
```
Comment (aboch) [inline engine/userguide/networking/default_network/ipv6.md]: Please restore this line, this was correct. `\80` means the first 80 bits do not change, so the first bit groups (16 bits each) in the address do not change: `2001:db8:23:42:1:`

**Comment by aboch** [comment]
Please squash the commits into one.
Besides that, looks good to me. Thanks.

**Pull request merged**